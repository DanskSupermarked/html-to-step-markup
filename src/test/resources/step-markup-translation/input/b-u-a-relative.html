<b><u><a href="/gd/100187664?intcmpid=vinStorkoeb">Ønsker du at købe mere end 20 gaveæsker? Kilk her</a></b></u>
