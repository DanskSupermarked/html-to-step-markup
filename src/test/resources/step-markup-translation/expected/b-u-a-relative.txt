<bold><underlined>Ønsker du at købe mere end 20 gaveæsker? Kilk her www.bilka.dk/gd/100187664?intcmpid=vinStorkoeb</underlined></bold>
